ethereum/core/src/test/java/org/hyperledger/besu/ethereum/bonsai/trielog/TrieLogPrunerTest.java

```diff
@@ -43,8 +43,10 @@ public class TrieLogPrunerTest {
   private BonsaiWorldStateKeyValueStorage worldState;
   private Blockchain blockchain;
 
+  @SuppressWarnings("BannedMethod")
   @BeforeEach
   public void setup() {
+    Configurator.setLevel(LogManager.getLogger(TrieLogPruner.class).getName(), Level.TRACE);
```

Comment:
why this one is needed for the test ?

Reply:
It's not _needed_ - purely convenience for comprehending the tests. I thought it was useful enough to leave in but happy to remove.